orchagent/qosorch.cpp

```diff
@@ -1019,13 +1034,15 @@ sai_object_id_t QosOrch::getSchedulerGroup(const Port &port, const sai_object_id
                 }
             }
 
-            m_scheduler_group_port_info[port.m_port_id].child_groups[ii] = std::move(child_groups);
+            scheduler_group_port_info.child_groups[ii] = std::move(child_groups);
         }
 
         for (const auto& child_group_id: child_groups_per_group)
         {
+            SWSS_LOG_INFO("Check child group %lx for port %s group %lx", child_group_id, port.m_alias.c_str(), group_id);
             if (child_group_id == queue_id)
             {
+                SWSS_LOG_INFO("Found group id %lx for port %s queue %lx", group_id, port.m_alias.c_str(), queue_id);
```

Comment:
There are too many logs added to code. Could you please revisit the logs? Also the compilation seems to be failing for armhf build. Please use the object ids as PRIx64. Refer existing logs

Reply:
Removed all the logs which were added but not related to a code change in this PR. For the rest, I tend to keep them.
Thank you for the comments on ARM building failure. Also fixed it.

Reply:
@prsunny can you please review it? comments fixed. thanks.